src/main/java/edu/rosehulman/covidtracer/model/PositiveCase.java

```diff
@@ -41,12 +41,12 @@
 	public PositiveCase() {
 	}
 
-	public PositiveCase(Long ID, Person person, Date d, Date d2, Date d3, boolean needsTransportation) {
+	public PositiveCase(Long ID, Person person, Date testAdministrationDate, Date startOfSymptoms, Date endOfQuarentineDate, boolean needsTransportation) {
```

Comment:
Please name this parameters testDate, symptomaticStartDate, and quarantineEndDate.